src/task-actions.ts

```diff
@@ -184,6 +185,14 @@ export async function starknetCompileAction(args: TaskArguments, hre: HardhatRun
 }
 
 export async function starknetDeployAction(args: TaskArguments, hre: HardhatRuntimeEnvironment) {
+    try {
+        const cacheHandled = await handleCache(args, hre);
```

Comment:
Here and in other Action implementations, what kind of error is being caught? If it's not something informative enough, it can just be caught in one place in handleCache and rethrown (so, thrown from the catch block).

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/106/commits/7cefdbcac0e358ea485983b4e184db810d5ea686

Reply:
As discussed, consider removing the try-catch block and making it a void function.